spec/shapes/bounds_spec.rb

```diff
@@ -18,13 +18,13 @@
 
   describe "with a bottom bound that is above the top" do
```

Comment:
Seems like you reused the same test title; is this testing the horizontal bounds?